pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1339,7 +1337,7 @@ def test_format_date_axis(self):
         xaxis = ax.get_xaxis()
         for line in xaxis.get_ticklabels():
             if len(line.get_text()) > 0:
-                assert line.get_rotation() == 30
+                assert line.get_rotation() == 0.0
```

Comment:
The label is not rotated visually.

Reply:
same, should it be?

Reply:
Same as the previous - on 3.5 `line.get_text()` is returning the empty string. In both 3.5 and 3.6, `line.get_rotation()` reports `0.0`, agreeing with the resulting plot.